python/ray/tune/result_grid.py

```diff
@@ -180,6 +180,7 @@ def _trial_to_result(self, trial: Trial) -> Result:
             checkpoint=checkpoint,
             metrics=trial.last_result.copy(),
             error=self._populate_exception(trial),
+            log_dir=trial.local_dir,
```

Comment:
Should this be `trial.logdir` instead?

Reply:
So the intention was to point to a place where tensorboard would be accesible from (so the `local_dir` of the trial). On the other hand, that information will also be contained in `trial.logdir` and accessible by getting its parent. I guess we could go with that to avoid confusion

Reply:
I have switched it to using `trial.logdir`